tests/test_api.py

```diff
@@ -38,6 +40,8 @@ class FailureHost:
             "Hostname mismatch, certificate is not valid for 'wrong.host.badssl.com'",
             # macOS
             "certificate name does not match",
+            # macOS with revocation checks
+            "certificates do not meet pinning requirements",
```

Comment:
@davisagli This is a strange message for the situation, makes me think of certificate pinning or macOS minimum requirements for certs. What's the full error message here if you have that handy?

Reply:
`SSLCertVerificationError('"*.badssl.com","R3","ISRG Root X1" certificates do not meet pinning requirements')`

Here is someone else configuring a revocation security policy similar to us, who noted that it's returning `errSecIncompleteCertRevocationCheck` = -67635 indicating that a revocation check could not be completed (which makes more sense), but that CFErrorCopyDescription is returning the wrong error message for that error code: https://developer.apple.com/forums/thread/706629

Reply:
Nice find!! That looks like exactly what we're experiencing.